mathics/main.py

```diff
@@ -246,6 +249,22 @@ def main():
         definitions, args.colors, want_readline=not(args.no_readline),
         want_completion=not(args.no_completion))
 
+    if args.initfile:
+        feeder = FileLineFeeder(args.initfile)
+        try:
+            while not feeder.empty():
+                evaluation = Evaluation(
+                    shell.definitions, output=TerminalOutput(shell), catch_interrupt=False)
+                query = evaluation.parse_feeder(feeder)
+                if query is None:
+                    continue
+                evaluation.evaluate(query, timeout=settings.TIMEOUT)
+        except (KeyboardInterrupt):
+            print('\nKeyboardInterrupt')
+        
+        definitions.set_line_no(0)
+
+
```

Comment:
Could you split this `initfile` stuff out into a separate PR please.

Reply:
Done